bgpd/bgp_bmp.c

```diff
@@ -2051,10 +2071,14 @@ DEFPY(bmp_connect,
       "Minimum connection retry interval\n"
       "Minimum connection retry interval (milliseconds)\n"
       "Maximum connection retry interval\n"
-      "Maximum connection retry interval (milliseconds)\n")
+      "Maximum connection retry interval (milliseconds)\n"
+      "Source address to use\n"
+      "Define an IPv6 source address\n"
+      "Define an IPv4 address\n")
```

Comment:
source address?

Reply:
a local address used to bind the tcp connection to the bmp client.

Reply:
I think the comment was meant as "why is it inconsistent with `IPv6 source address` but `IPv4 address`".

Reply:
OK i take that point into account